src/main/resources/assets/nei/guidfilters.cfg

```diff
@@ -0,0 +1,56 @@
+AdvancedSolarPanel:hybrid_solar_helmet,Damage
```

Comment:
The file is in the jar, but you can also be specified it in the config to overwrite